src/main/java/seedu/address/MainApp.java

```diff
@@ -142,7 +143,7 @@ protected Config initConfig(Path configFilePath) {
      */
     protected UserPrefs initPrefs(UserPrefsStorage storage) {
         Path prefsFilePath = storage.getUserPrefsFilePath();
-        logger.info("Using preference file : " + prefsFilePath);
+        logger.fine("Using preference file : " + prefsFilePath);
 
         UserPrefs initializedPrefs;
```

Comment:
My opinion on this is that users should only see logs when a new file is created or some error occurred while trying to load the file. When the file is loaded correctly from the path, I feel that the logs here aren't actually useful for the users since it's expected to be loaded correctly for the user which was why I opt to use fine for logs that say "Using xxx file: <file path>".

The argument to use info logs might be if the user wants to edit/modify the related files manually so it might be useful to show file path location with info but based on the [ug](https://se-education.org/addressbook-level3/UserGuide.html#saving-the-data), so there isn't really a need to display the file locations.

Reply:
That is a fair point. But the UG mentions the data file only. 
Also, the log is not for errors/warnings only -- if we wanted that, we should have set the default log level to WARN or ERROR. At INFO level, it should show important activities that are happening.